src/stan-users-guide/user-functions.qmd

```diff
@@ -287,6 +287,32 @@ transformed parameters {
 }
 ```
 
+## Functions implementing change-of-variable adjustments
+
+Functions whose names end in `_jacobian` can use the
+`jacobian +=` statement. This can be used to implement a custom
+change of variables for arbitrary parameters.
+
+For example, here is a program which re-creates the built-in
+`<upper=x>` transform on real numbers:
+
+```stan
+functions {
+  real upper_bound_jacobian(real x, real ub) {
+    jacobian += x;
+    return ub - exp(x);
+  }
+}
+data {
+  real ub;
+}
+parameters {
+  real b_raw;
+}
+transformed parameters {
+  real b = upper_bound_jacobian(b_raw, ub);
+}
+```
```

Comment:
This is a duplicate and I'd like a sampling statement for `b` in the model block.

Reply:
Duplication is intentional here as it's in a very different part of the docs, but I can do something unique if you prefer

Reply:
Understood.  I'd change it up just for variety, but I can see the motivation for duplicating, so up to you.